pkg/controller/logcollector/logcollector_controller.go

```diff
@@ -429,6 +428,18 @@ func (r *ReconcileLogCollector) Reconcile(ctx context.Context, request reconcile
 		}
 	}
 
+	var syslogCredential *render.SysLogCredential
```

Comment:
I think this is not correct:
- A secret is used, but the contents are a certificate only. This is not secret information. I think the certificate could be part of the LogCollector Spec, much like how you can add certificatePEM for Installation.Spec.CertificateManagement. @tmjd WDYT?
- For this reason `syslogCredential` seems to be the wrong variable name as well.

Reply:
It looks like this has been discussed in the design document, I think @caseydavenport would be better to consult here since he has been reviewing the design doc.